### PR TITLE
docs: CEL Expressions for Access Logging

### PR DESCRIPTION
**Pull request by zirain**
fix: https://github.com/envoyproxy/gateway/issues/3734

### CODE REVIEW

File: site/content/en/latest/tasks/observability/proxy-accesslog.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+```shell
+LOKI_IP=$(kubectl get svc loki -n monitoring -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+```
+
+## Default Access Log
+
+If custom format string is not specified, Envoy Gateway uses the following default format:
+
+```json
```
Comment (shawnh2) [inline site/content/en/latest/tasks/observability/proxy-accesslog.md:28]: can we prettify this json ? it's not very readable

File: site/content/en/latest/tasks/observability/proxy-accesslog.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+```shell
+curl -s "http://$LOKI_IP:3100/loki/api/v1/query_range" --data-urlencode "query={job=\"fluentbit\"}" | jq '.data.result[0].values'
+```
+
+## Disable Access Log
+
+If you want to disable it, set the `telemetry.accesslog.disable` to `true` in the `EnvoyProxy` CRD.
+
+```shell
+kubectl apply -f - <<EOF
```
Comment (shawnh2) [inline site/content/en/latest/tasks/observability/proxy-accesslog.md:46]: we should have two tabs for this, one is apply from cmd, the other is apply from file.

File: site/content/en/latest/tasks/observability/proxy-accesslog.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
+---
+title: "Proxy Access Log"
```
Comment (shawnh2) [inline site/content/en/latest/tasks/observability/proxy-accesslog.md:2]: different terms are used in this doc, like `access log` & `access logs`, should be consistent, prefer `access logs`

File: site/content/en/latest/tasks/observability/proxy-accesslog.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+```
+
+Verify logs from loki:
+
+```shell
+curl -s "http://$LOKI_IP:3100/loki/api/v1/query_range" --data-urlencode "query={exporter=\"OTLP\"}" | jq '.data.result[0].values'
+```
+
+
+### Additional Metadata
```
Comment (shawnh2) [inline site/content/en/latest/tasks/observability/proxy-accesslog.md:170]: should this be a level two title ?

**Review by shawnh2 — APPROVED** [review]

File: site/content/en/latest/tasks/observability/proxy-accesslog.md (hunk 2 above)
Comment (zirain) [reply site/content/en/latest/tasks/observability/proxy-accesslog.md:71]: it's fine now.

**Review by shawnh2 — APPROVED** [review]
LGTM!

**Comment by zhaohuabing** [comment]
Should we also mention the metadata(httproute, gateway) in the data plane log?

**Comment by zirain** [comment]
> Should we also mention the metadata(httproute, gateway) in the envoy log?

that should be in another pr.

**Review by zhaohuabing — APPROVED** [review]

**Pull request merged**